packages/lesswrong/components/common/LWPopper.tsx

```diff
@@ -81,7 +84,7 @@ const LWPopper = ({
     }
   ];
 
-  const { styles, attributes } = usePopper(anchorEl, popperElement, {
+  const { styles, attributes, update } = usePopper(anchorEl, popperElement, {
```

Comment:
When the styles on the anchor element change, the popper doesn't move by default. I'm exposing this update function to callers so they can manually trigger it to recalculate it's position